roles/etcd/tasks/main.yml

```diff
@@ -37,6 +37,12 @@
     mode: 0700
   when: etcd_is_thirdparty | bool
 
+- name: Create configuration file in sysconfig
+  file:
+    path: /etc/sysconfig/etcd
+    state: touch
+  when: etcd_is_thirdparty | bool
```

Comment:
Once https://github.com/openshift/openshift-ansible/pull/4217 is merged, you can use:
```yaml
when: openshift.common.etcd_runtime == "host" or openshift.common.etcd_runtime == "docker"
```
So it is explicitly sad who is consumer of the file.